rfd/0046-database-access-config.md

```diff
@@ -64,14 +66,269 @@ $ clusterawsadm bootstrap iam print-policy --document AWSIAMManagedPolicyControl
+  #   # Supported protocols for Redshift: "postgres" or "mysql"
+  #   protocol: postgres
+  #   # Database connection endpoint. Must be reachable from Database service.
+  #   uri: redshift-cluster-example-1.abcdefghijklmnop.us-west-1.redshift.amazonaws.com:5439
+  #   # AWS specific configuration.
+  #   aws:
+  #     # Region the database is deployed in.
+  #     region: us-west-1
+  #     # Redshift specific configuration.
+  #     redshift:
+  #       # Redshift Cluster ID.
+  #       cluster_id: redshift-cluster-example-1
+  # # Self-hosted static configuration.
+  # - name: self-hosted
+  #   description: Self-hosted database configuration.
+  #   # Supported protocols for self-hosted: postgres, mysql, mongodb, cockroachdb.
+  #   protocol: postgres
+  #   # Database connection endpoint. Must be reachable from Database service.
+  #   uri: database.example.com:5432
+auth_service:
+  enabled: "no"
+ssh_service:
+  enabled: "no"
+proxy_service:
+  enabled: "no"
+```
+
+Similarly to configure command, it will also support writing the configuration
+directly to a file.
+
+```bash
+# write to the default file location.
+$ teleport db configure create --output file
+
+Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
+```
+
+```bash
+# write to the provided file location.
+$ teleport db configure create --output file:///teleport.yaml
+
+Wrote config to file "/teleport.yaml". Now you can start the server. Happy Teleporting!
+```
+
+Configure a single database.
+
+```bash
+# generates a configuration for a Postgres database.
+$ teleport db configure create \
+   --token=/tmp/token \
+   --auth-server=localhost:3025 \
+   --name=sample-db \
+   --protocol=postgres \
+   --uri=postgres://localhost:5432
+```
+
+### Bootstrap permissions command
 
-The "configure" command will provide subcommands for configuring database agents
-in different environments. This RFD focuses on `aws` subcommands that deal with
-configuring IAM for agents that proxy AWS-hosted databases (RDS, Aurora,
-Redshift).
+Subcommand will bootstrap the necessary configuration for the database agent. It
+reads the provided agent configuration to determine what will be bootstrapped.
+
+```bash
+$ teleport db configure bootstrap [-c=,--config=] [--mode=] [--policy-name=] [--attach-to=] [--confirm]
+```
+
+| Flag            | Description |
+| --------------- | ----------- |
+| `-c, --config`  | Path to the database agent configuration file. Default: "/etc/teleport.yaml". |
+| `--mode`        | How the bootstrap will be performed. “automatic” will make the changes directly in the cloud provider. "manual" will print the instructions to complete the configuration. Default: "manual". |
+| `--policy-name` | Name of the Teleport Database agent policy. Default: "". |
+| `--attach-to`   | Target that the Teleport policy is going to be attached to. If empty, the policy won't be attached to anything. |
```

Comment:
Shouldn't the behavior rather be, if this is empty then we'll try to attach to the AWS identity you're currently running as (i.e. your AWS user/role)?